common/views/components/CopyUrl/CopyUrl.js

```diff
@@ -23,7 +23,7 @@ const CopyUrl = ({id, url}: Props) => (
       data-copy-text={url}
       data-track-event={`{"category": "component", "action": "copy-url:click", "label": "id:${id}"}`}
       className={`${spacing({s: 2}, {margin: ['top']})} ${font({s: 'HNM5', m: 'HNM4'})} btn btn--light is-hidden js-copy-url pointer`}>
-      <Icon name='check' extraClasses='icon--black is-hidden' />
+      <Icon name='check' extraClasses={['icon--black', 'is-hidden']} />
```

Comment:
Wondering if we should make extraClasses a string so it's consistent with other components

Reply:
Damn, just saw this, happy either way... It's used all over the place, so a big find / replace will do it.

Reply:
Also stops us guessing what it might be.